lerna.json

```diff
@@ -1,11 +1,6 @@
 {
-  "packages": [
-    "packages/*",
-    "apps/*",
-    "scripts",
-    "packages/fluentui/*"
-  ],
+  "packages": ["apps/*", "packages/*", "scripts", "packages/fluentui/*"],
   "npmClient": "yarn",
   "useWorkspaces": true,
-  "version": "0.0.0"
+  "version": "0.44.0"
```

Comment:
Wish there was a way to comment on this file about why we're using this version

Reply:
i hate json file - wish this was a .js file

Reply:
:'(